internal/preprocessing/preprocessing.go

```diff
@@ -24,9 +28,25 @@ type WorkflowParams struct {
 	RelativePath string
 }
 
+type Outcome int
+
+const (
+	OutcomeSuccess Outcome = iota
+	OutcomeSystemError
+	OutcomeContentError
+)
+
 type WorkflowResult struct {
+	// Outcome is an integer indicating if the workflow completed successfully,
+	// or with errors.
+	Outcome Outcome
+
 	// Relative path to the shared path.
 	RelativePath string
+
+	// IngestTasks is a log of the preservation tasks performed by
+	// preprocessing.
+	IngestTasks []ingest.Task
```

Comment:
How is this mapped to https://github.com/artefactual-sdps/preprocessing-sfa/blob/main/internal/workflow/preprocessing.go#L35?

Reply:
Hmm, I don't know how that would map.  Maybe I broke the integration when I changed the name? I'll switch back to "PreservationTasks".